src/styles/components/_logo.scss

```diff
@@ -1,4 +1,5 @@
 .logo {
+  z-index: $zindex-logo;
```

Comment:
Another quick question, do we really need to set z-index for this?

Reply:
No need krub.